### DISCUSSION TITLE
I want all 16 GoPros to be in sync the same way

### DISCUSSION THREAD

**Discussion by wonder21c** [gopro/labs#1010] [Q&A]
![KakaoTalk_20240802_170708979](https://github.com/user-attachments/assets/535e30b6-fddd-46b1-b839-b943ab587e61)
I have 16 GoPros. I want these devices to all start at the same time with the same timecode. I think a difference of about 0.5 to 1 frame would be okay.
However, even if you check the "set date and time" option with the QR Creator app, there is quite a bit of frame difference when shooting the video. How can I get all 16 GoPros to start with the same timecode?

**Comment by dnewman-gpsw** [comment]
Timecode should be within a frame at 30fps. That is not the same as starting at the same time.  Timecode is there so that can be resync'd in post.  When using a command like ```!10:00S```  <- start at 10am, this is not frame accurate, likely within a second.    

For high precision multi-cam for gaussian splatting or other computational imaging.  It is best to over-crank the cameras, shooting 4K120 or 2.7k240, that using audio or a flash gun to help more precisely sync the cameras in post.  It all does take work.

**Reply by wonder21c** [reply]
Is there a way to set the time in more detail than !10:00S?

**Reply by dnewman-gpsw** [reply]
To the minute is supported, and it is accurate to within a second.  Having seconds digits would not improve the precision.

**Comment by BonN2Tech** [comment]
What do you want to do with the sync'd videos? For example, do you only want them to all playback simultaneously on one or more screens and in sync for presentation and review or do multicam editing between videos?
